src/server/DeviceServer_v2.js

```diff
@@ -299,21 +299,21 @@ class DeviceServer {
     }
 
     // TODO figure this out
-    if (lowerEventName.indexOf('spark/device/safemode') === 0) {
-      const token = device.sendMessage('Describe');
-      const systemMessage = await device.listenFor(
-        'DescribeReturn',
-        null,
-        token,
-      );
-
-      if (global.api) {
-        global.api.safeMode(
-          deviceID,
-          systemMessage.getPayload().toString(),
-        );
-      }
-    }
+    // if (lowerEventName.indexOf('spark/device/safemode') === 0) {
```

Comment:
Ahh... I started working on the claim process from the cli but there are bugs with the windows implementation where Windows 10 will not connect to the device sometimes.

I'd love to get the safemode and version working. We aren't saving the correct version right now in our implementation.